packages/sdk/tests/key-management.unit.test.ts

```
@@ -0,0 +1,276 @@
+    constructor(mnemonicWords?: string[]) {
+        if (mnemonicWords) {
+            this.hdNode = HDKey.fromMnemonic(mnemonicWords);
+        } else {
+            // Generate new mnemonic
+            const mnemonicWords = Mnemonic.of();
+            this.hdNode = HDKey.fromMnemonic(mnemonicWords);
+        }
+    }
+    
+    // Create account from HD wallet
+    createAccount(index: number = 0) {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)